servicesCrud/src/services/manager.ts

```diff
@@ -0,0 +1,31 @@
+import { ServiceRepository } from './repository';
+import { IService } from './interface';
+import { ServiceNotFound } from '../utils/errors/services';
+
+export class ServiceManager {
+  static async create(newservice: IService): Promise<IService> {
+    return ServiceRepository.create(newservice);
+  }
+
+  static async updateServiceById(newservice: Partial<IService>, id: string): Promise<IService | null> {
+    return ServiceRepository.updateServiceById(newservice, id);
+  }
+
+  static async getServiceById(serviceId: string): Promise<IService | null> {
+    const service = await ServiceRepository.getServiceById(serviceId);
+    if (!service) throw new ServiceNotFound();
+    return service;
```

Comment:
- relevant in other functions

after implemeting `orFail` this isn't needed anymore and you can do
```suggestion
    return ServiceRepository.getServiceById(serviceId);
```